scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -94,6 +94,15 @@ SG_ DUMMY_AIR_SHUTDOWN : 0|1@1+ (1,0) [0|1] "" DEBUG
 BO_ 208 DCM_MOTOR_SHUTDOWN_ERRORS: 8 DCM
 SG_ DUMMY_MOTOR_SHUTDOWN : 0|1@1+ (1,0) [0|1] "" DEBUG
 
+BO_ 209 DCM_ACCELERATION_X: 4 DCM
+SG_ ACCELERATION_X : 0|32@1+ (0.01,-30.00) [-30.00|30.00] "m/s^2" DEBUG
```

Comment:
These shouldn't be changed

```suggestion
SG_ ACCELERATION_X : 0|32@1+ (1,0) [-30.00|30.00] "m/s^2" DEBUG
```

Reply:
ok I changed it back. Is there a case when I should use them? What are they for?

Reply:
I think this scales your value that you're broadcasting.

Reply:
![image](https://user-images.githubusercontent.com/32230980/97255864-c920bd80-17ce-11eb-8906-444181766c62.png)

Reply:
ok, makes sense